tests/ui/lang/consts/nested-ref.stderr

```diff
@@ -0,0 +1,6 @@
+warning: function `nested_ref::deep_load` has `dont_inline` attribute, but need to be inlined because it has illegal argument or return types
+
+warning: function `nested_ref::deep_transpose` has `dont_inline` attribute, but need to be inlined because it has illegal argument or return types
+
+warning: 2 warnings emitted
+
```

Comment:
You'll need to `git pull` the new commit locally then run `cargo compiletest --bless` again to update this `.stderr` file.